base/abstractarray.jl

```diff
@@ -1830,23 +1830,22 @@ dimensions for every new input array and putting zero blocks elsewhere. For exam
 `matrices[1]`, `matrices[2]`, ... as diagonal blocks and matching zero blocks away from the
 diagonal.
 
-julia> cat(a, b, dims=3)
-2×2×4 Array{Int64, 3}:
-[:, :, 1] =
- 1  2
- 3  4
-
-[:, :, 2] =
- 5  6
- 7  8
+See also [`hcat`](@ref), [`vcat`](@ref), [`hvcat`](@ref), [`repeat`](@ref).
 
-[:, :, 3] =
-  9  10
- 11  12
-
-[:, :, 4] =
- 13  14
- 15  16
+# Examples
+```jldoctest
+julia> cat([1 2; 3 4], [pi, pi], fill(10, 2,3,1); dims=2)
+2×6×1 Array{Float64, 3}:
+[:, :, 1] =
+ 1.0  2.0  3.14159  10.0  10.0  10.0
+ 3.0  4.0  3.14159  10.0  10.0  10.0
+ 
```

Comment:
```suggestion

```

Reply:
Sorry about the space, but won't it need a newline here?

Reply:
Oops! :)

Reply:
Let's hope I didn't screw it up this time...